Compilation-policy/README.md

```diff
@@ -40,7 +40,9 @@ For implementors:
   (= (cleavir-policy:optimize-value optimize 'safety) 3))
 ```
 
-3. Make sure your client respects the `OPTIMIZE-INFO` protocol
-   in `cleavir-environment`. You can use `COMPUTE-POLICY` to
-   compute policies from optimize declarations, but you should
-   avoid doing this on every `OPTIMIZE-INFO` call.
+<!-- FIXME(paul): Does it really have to respect the protocol?
+     In what way is it used here? -->
```

Comment:
this comment makes less sense in a trucler context, i think. it might be enough to say that we use trucler to get optimize info, so if the client wants policies to make sense, they need to implement trucler properly. `describe-optimize` no longer  has to return policy info (though it probably should to avoid recomputation)

Maybe something like...

> Make sure your client respects the `OPTIMIZE-DESCRIPTION` protocol in Trucler. Cleavir expects additional policy information. By default it will compute a policy anew each time it is needed, but it is recommended that clients cache policy information alongside optimize information to avoid this. `CLEAVIR-CST-TO-AST:POLICY` should be specialized to retrieve from this cache rather than do computation.